pkg/platform/runtime/runtime.go

```diff
@@ -90,7 +89,6 @@ func New(target setup.Targeter, an analytics.Dispatcher, svcm *model.SvcModel, a
 	recordAttempt(an, target)
 	an.Event(anaConsts.CatRuntimeDebug, anaConsts.ActRuntimeStart, &dimensions.Values{
 		Trigger:          ptr.To(target.Trigger().String()),
-		Headless:         ptr.To(strconv.FormatBool(target.Headless())),
```

Comment:
Thinking out loud: do we want to deprecate this dimension? We've done so in the past for other dimensions, but I don't know exactly what that entails.

Reply:
We've deprecated events in the past but I'm not sure about dimensions... I'll see what I can find. We want to keep the dimension around for now as there may be headless projects out in the wild.